SeleniumPracticalExercise/PageObjects/LoginPage.cs

```diff
@@ -14,8 +14,10 @@ public LoginPage(IWebDriver driver) : base(driver)
         }
 
         /// <summary>
-        /// Enter (SendKeys) for username and password then Click Submit button
+        /// Enter username and password 
```

Comment:
It does more than just enters the username and password... it logs in. Should be something more like, "Logs in using the provided username and password."